app/services/permission.rb

```diff
@@ -56,4 +58,15 @@ def guest_user_permissions
     return true if controller == "welcome" && action.in?(%w(index))
   end
 
+  def pending_user_permissions
+    return true if controller == "sessions"
+    return true if controller == "candidates"  && action.in?(%w(index show))
+    return true if controller == "cart"
+    return true if controller == "donations"
+    return true if controller == "issues"  && action.in?(%w(index show))
+    return true if controller == "users"  && action.in?(%w(new create))
```

Comment:
Unnecessary spacing detected.<br>Operator `&&` should be surrounded with a single space.